stylesheets/commons/Brackets.less

```diff
@@ -485,7 +485,6 @@
 		overflow: auto;
 		bottom: 16%;
 		left: 1px !important;
-		max-height: 60vh;
```

Comment:
i don't see any usage for `.brkts-match-info-popup-sc-ffa` anymore
imo just kick it entirely (lines 483-496)